scripts/test/MultiPlotting/MultiPlottingContext_test.py

```diff
@@ -6,10 +6,11 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 import unittest
 
+from mantid.simpleapi import *
```

Comment:
I've never quite been certain on this, is it our standard to import everything from the simpleapi as standard? I see this done very commonly but have always personally preferred importing just the required algorithms so it is easier to track down where they have come from in the code.

Reply:
not needed, so deleted it